Sources/gg.datagram.web-requests.sdPlugin/app.js

```diff
@@ -13,13 +13,43 @@ function connected(jsn) {
  *       url?: string,
  *       method?: string,
  *       body?: string|null,
+ *       contentType?: string,
+ *       headers?: string[]|null,
+ *       body?: string|null,
  *     }
  *   },
  * }} data
  */
 function sendHttp(data) {
-    const { url, method, body } = data.payload.settings;
-    log('sendHttp', { url, method, body });
+    const { url, method, contentType, headers, body } = data.payload.settings;
+    log('sendHttp', { url, method, contentType, headers, body });
+
+    let defaultHeaders = {
+        'Content-Type': typeof contentType !== 'undefined' && contentType ? contentType : 'text/plain'
```

Comment:
If `contentType` isn't provided, `defaultHeaders` should be an empty object.

Also, FYI, the code here could be written much more succinctly as `contentType || 'text/plain'`